### DISCUSSION TITLE
[Modification] - Reduce line height of some type styles

### DISCUSSION THREAD

**Discussion by alexcanning** [Legal-and-General/canopy#1174] [Canopy requests]
**Type of request**
Modification

**Component, pattern, template, helper or foundation?**
Foundation

**Nature of request**
The current type styles `lg-font-size-5`, `lg-font-size-6` and `lg-font-size-7` have been found to have line heights that are too spacious. I would like the following style's line heights reduced accordingly:

| **Style name**     | **Type size (LG)** | **New line height (LG)** | **Type size (SM)** | **New line height (SM)** |
| ------------------ | ------------------ | ------------------------ | ------------------ | ------------------------ |
| **lg-font-size-7** | 56                 | 68                       | 52                 | 64                       |
| **lg-font-size-6** | 48                 | 60                       | 44                 | 56                       |
| **lg-font-size-5** | 40                 | 52                       | 36                 | 48                       |

These new properties have been tested and refined by the L&G D&E design team.

Assume this would be a breaking change.

**Comment by elenagarrone** [comment]
PR here: https://github.com/Legal-and-General/canopy/pull/1222

**Comment by elenagarrone** [comment]
The change is now into the `next` branch.
